src/nlist.jl

```diff
@@ -4,18 +4,9 @@
 # matscipy neighbourlist functionality
 ############################################################
 
-include("MatSciPy.jl")
-
-function neighbourlist(at::ASEAtoms, cutoff::Float64)::MatSciPy.NeighbourList
-   # TODO: also recompute if rcut is different !!!!!
-   # if no previous neighbourlist is available, compute a new one
-   # if !has_transient(at, (:nlist, cutoff)) || recompute
-   #    # this nlist will be destroyed as soon as positions change
-   #    set_transient!(at, (:nlist, cutoff), MatSciPy.NeighbourList(at, cutoff))
-   # end
-   # return get_transient(at, (:nlist, cutoff))
-   return MatSciPy.NeighbourList(at, cutoff)
-end
+# include("MatSciPy.jl")
+using MolSimPy
+using .MatSciPy: neighbourlist
```

Comment:
why is this line still here? Should this not import from `MolSimPy`

Reply:
yeah, sorry.